samples/app-dev/Flow-Design-Concepts/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -26,20 +28,27 @@ The anyOf keyword allows for validation against multiple schemas. If the data va
 
 7. Click Next. If you had not selected a trigger in the previous dialog, the flows associated with that trigger are displayed. You have the option to select one or more of these flows such that the flows get imported as blank flows that are not attached to any trigger. By default, all flows are selected. Clear the check box for the flows that you do not want to import. If your flow(s) have subflows, and you select only the main flow but do not select the subflow, the main flow gets imported without the subflow. Click Next.
 
-8. Click on *oneOfsampleflow* and click on mapper activity,navigate to input tab and expand all configurations.Click on oneOf schema and select the radio button corrosponding to needed schema.Save the configuration and verify that in mapper input tab,properties of selected schema are appeared in activity inputs.
-![oneOf configuration](../../import-screenshots/oneOf_configuration.png)
+8. Update the service endpoints url's for allOf, oneOf and anyOf in app Properties window for invoke app. Validate, Push and Scale the apps
```

Comment:
Instead of updating app properties here, we should push the invoking the app first and then update the URLs in the Environment controls tab -> App properties